2016-04-28 22:29:38,199 - streamparse.storm.component - ERROR - Exception in WordCounter.run()
Traceback (most recent call last):
  File "/opt/py27environment/lib/python2.7/site-packages/streamparse/storm/component.py", line 487, in run
    self.initialize(storm_conf, context)
  File "bolts/wordcount.py", line 24, in initialize
    cur.execute("""CREATE TABLE tweetwordcount (word TEXT PRIMARY KEY   NOT NULL, count INT NOT NULL)""")
ProgrammingError: relation "tweetwordcount" already exists

